UnitTests/Controllers/Bundle Support/SPBundleCommandRunnerTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  SPBundleCommandRunnerTests.swift
+//  Unit Tests
+//
+//  Created by Christopher Jensen-Reimann on 10/31/21.
+//  Copyright © 2021 Christopher Jensen-Reimann.
+//
+//  Permission is hereby granted, free of charge, to any person
+//  obtaining a copy of this software and associated documentation
+//  files (the "Software"), to deal in the Software without
+//  restriction, including without limitation the rights to use,
+//  copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the
+//  Software is furnished to do so, subject to the following
+//  conditions:
+//
+//  The above copyright notice and this permission notice shall be
+//  included in all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+//  OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+//  HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+//  WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+//  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+//  OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import XCTest
+
```

Comment:
You might need @testable import SequealAce or similar?

Reply:
![Screen Shot 2021-11-01 at 10 23 46 AM](https://user-images.githubusercontent.com/930039/139687203-218c3bef-8cb4-4b58-ba66-08eae7155880.png)
or else

![Screen Shot 2021-11-01 at 10 50 12 AM](https://user-images.githubusercontent.com/930039/139691115-d76c5c13-f1fa-4e17-bac1-706558f82c0f.png)

Reply:
@Kaspik or @Jason-Morcos Do either of you have experience debugging unit test build issues? 

This is usually the point where I throw up my hands and skip unit testing in XCode, because it does something dumb. Like here where it can't find my class when it goes to link the unit test executable. The test compiled, so it clearly found `SABundleRunner.h` but somehow when things get linked. See #1277 as very self-contained example of this behavior.

Reply:
You need to add the file to the bridging header, don't you?

Reply:
Unfortunately, I don't have much experience with Xcode Unit Tests. I've always found them too cumbersome/temperamental to bother with 😬 
@Kaspik is definitely the better expert on these things, likely a few other Obj-C experts like @jamesstout too (if he's available).